src/libshadertrap/src/command_assert_equal.cc

```diff
@@ -20,6 +20,17 @@
 
 namespace shadertrap {
 
+CommandAssertEqual::CommandAssertEqual(
+    std::unique_ptr<Token> start_token, bool arguments_are_renderbuffers,
+    std::unique_ptr<Token> argument_identifier_1,
+    std::unique_ptr<Token> argument_identifier_2,
+    std::vector<FormatEntry> format_entries)
+    : Command(std::move(start_token)),
+      arguments_are_renderbuffers_(arguments_are_renderbuffers),
+      argument_identifier_1_(std::move(argument_identifier_1)),
+      argument_identifier_2_(std::move(argument_identifier_2)),
+      format_entries_(std::move(format_entries)) {}
+
```

Comment:
It only makes sense for `format_entries` to be present when we are comparing data buffers, not renderbuffers (renderbuffers are basically images).

I therefore propose that we change things to that:

- the `arguments_are_renderbuffers` parameter is dropped
- the constructor that takes `format_entries` is invoked if and only if the buffers being compared are data buffers (so the `arguments_are_renderbuffers` field gets set to `false` by this constructor
- the constructor without `format_entries` is invoked if and only if the buffers being compared are renderbuffers (so that the boolean gets set to `true`)